Tests/src/PublicAPITests/SpaceSystemTests.cpp

```diff
@@ -279,6 +279,24 @@ void GetSpacesMetadata(::SpaceSystem* SpaceSystem, const Array<String>& SpaceIds
 	OutMetadata = Result.GetMetadata();
 }
 
+void GetSpaceTags(::SpaceSystem* SpaceSystem, const String& SpaceId, Array<String>& OutTags)
```

Comment:
[Consider] I'd strongly recommend calling these like `GetAndAssertSpaceTags`, or something with assert/test/check in the title, so people can understand the test flow from the method names.

Reply:
done in ac6f3d3006b386a3f496bff7fee3023e0ead9334